### ISSUE TITLE
Remove empty-slot facilities

### ISSUE THREAD

**Issue by pont-us** [pont-us/PuffinPlot#195]
Empty-slot corrections need to be rethought and reimplemented, and there's no time to do that for version 1.0. The best plan is to remove at least the menu items (to avoid confusing users) and probably the code too (since it's not currently used, and since it would probably need re-writing and testing before being useful); it'll be in the repo if I ever need it again  anyway.

**Comment by pont-us** [comment]
```
Bugs Everywhere data:
Created at: 2011-12-10 23:30:30
Status: fixed
Severity: minor
UUID: c84b8dbf-8177-4b89-9d27-51e0c811337c
Short name: a88/c84
Ditz bug index: 193
```

**Comment by pont-us** [comment]
2011-12-11 00:23:52

I've just removed the menu items, but left the underlying code. It's unlikely that tray corrections will become an on-the-fly item again (with all the complex measurement protocols I've introduced), but it's possible that a sensible implementation of the empty-slot correction would apply it on-the-fly. In that case, there's no reason not to let the user toggle it. So I'm leaving a code skeleton to allow that, for now.

UUID: 4fae4824-dfa0-4f52-b747-019e4b948dfe
